src/components/molecules/NavBar/playa.scss

```diff
@@ -16,7 +16,7 @@ $sand: #937c63;
 
 .navbar.navbar_playa {
   position: fixed;
-  z-index: 100;
+  z-index: 301;
```

Comment:
Any reason for `301` specifically... let's move this zindex to a variable, and aim to move more later so we have them all in one place.